CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ and this project adheres to
 
 ## [Unreleased]
 
+### Added
+
+- Added `totalMemory`, `totalDiskSpace`, and `lastOnline` to `Device` entity
```

Comment:
You don't have to change this, but I've been using a table for new properties. See: https://github.com/JupiterOne/graph-google-cloud/blob/main/CHANGELOG.md#added-1

Reply:
@austinkelleher I like that, updated to do that.